tests/language_specific/english/test_english_language_generator.py

```diff
@@ -245,6 +246,132 @@ def test_dad_put_a_cookie_in_a_box_using_i():
     ).as_token_sequence() == ("I", "put", "a", "cookie", "in", "a", "box")
 
 
+def test_dad_put_a_cookie_in_a_box_using_you():
```

Comment:
We also want an test for the simple case of "Dad has a cookie" with no other action going on.